### ISSUE TITLE
In Document Viewer: Sidebar section headers (e.g. Categories, Issues) cannot be accessed via keyboard – users should be able to tab to these elements and operate them via keyboard

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12145]
The Categories and Issue tags buttons cannot be accessed with a keyboard. These elements can only be clicked with a mouse. (New: 08/27/2019) the only button that looks to be actionable is the filter button which are not accessible via the keyboard

**Comment by ajspotts** [comment]
@alisan16 I was unable to recreate the problem listed in the ticket.  Utilizing JAWS I was able to successfully navigate with the keyboard to both the Categories and Issue Tags and interact.  
Here is the interaction with the Categories filter:

![ezgif com-crop](https://user-images.githubusercontent.com/18618189/75271762-bb8fdf80-57ca-11ea-95da-f0913de576d6.gif)

Here is the interaction with the Issue Tags filter:

![Feb-25-2020 12-43-22](https://user-images.githubusercontent.com/18618189/75272800-8f755e00-57cc-11ea-876e-3aa547cbe17b.gif)

**Comment by alisan16** [comment]
@ajspotts This ticket is actually about Caseflow Reader, which is the document viewer that opens when you click on one of the documents. The categories/issues section on the right hand side bar:

<img width="1332" alt="Screen Shot 2020-02-25 at 4 46 28 PM" src="https://user-images.githubusercontent.com/1198106/75290493-6e256980-57ee-11ea-9a8c-ee0ee793114f.png">

**Comment by ajspotts** [comment]
@alisan16 I was also able to navigate the Reader sidebar Categories and Issue tags sections using JAWS without issue.

![Feb-26-2020 14-57-51](https://user-images.githubusercontent.com/18618189/75382413-72ae5880-58a8-11ea-872a-37174f162347.gif)

**Comment by alisan16** [comment]
Thanks for checking @ajspotts! Perhaps one of our other fixes already solved this problem too. I'm closing this ticket out and going to follow up with @carodew to see if they have issues using JAWS to access this.
